### PR TITLE
Metrics report stored locally

### PR DESCRIPTION
**Pull request by michalwski**
This PR stores the last sent report to a JSON file.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[7838.1](https://travis-ci.org/esl/MongooseIM/jobs/642810540) / Erlang 22.0 / small_tests / ab5f6c3a0f50b49d4e4d03788c4942047aaf580a
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2612/7838/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2612/7838/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-28_10.06.18/lib.mongooseim.logs/run.2020-01-28_10.06.19/suite.log.html)

File: big_tests/tests/service_mongoose_system_metrics_SUITE.erl [hunk 1]
```diff
@@ -255,6 +252,17 @@ system_metrics_are_reported_to_configurable_google_analytics(_Config) ->
     mongoose_helper:wait_until(fun events_are_reported_to_additional_tracking_id/0, true),
     mongoose_helper:wait_until(fun events_are_reported_to_configurable_tracking_id/0, true).
 
+system_metrcis_are_reported_to_a_json_file(_Config) ->
```
Comment (NelsonVides) [inline big_tests/tests/service_mongoose_system_metrics_SUITE.erl]:
```suggestion
system_metrics_are_reported_to_a_json_file(_Config) ->
```

File: big_tests/tests/service_mongoose_system_metrics_SUITE.erl [hunk 2]
```diff
@@ -79,6 +75,7 @@ all() ->
      system_metrics_are_reported_to_google_analytics_when_mim_starts,
      system_metrics_are_reported_to_additional_google_analytics,
      system_metrics_are_reported_to_configurable_google_analytics,
+     system_metrcis_are_reported_to_a_json_file,
```
Comment (NelsonVides) [inline big_tests/tests/service_mongoose_system_metrics_SUITE.erl]:
```suggestion
     system_metrics_are_reported_to_a_json_file,
```

**Comment by michalwski** [comment]
I had the same dilemma.. I decided to go the simplest way for now but can introduce the behaviour if you want.

**Comment by mongoose-im** [comment]
[7841.1](https://travis-ci.org/esl/MongooseIM/jobs/642833768) / Erlang 22.0 / small_tests / 20f92e38a1eec89edbc96f9da0c91dfa006e1376
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2612/7841/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2612/7841/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-28_13.25.58/lib.mongooseim.logs/run.2020-01-28_13.25.58/suite.log.html)

**Pull request merged**